### PR TITLE
Update repo documentation

### PR DESCRIPTION
**Pull request by jenniferboedker**

### CODE REVIEW

File: .github/release.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+    authors:
+      - JohnnyQ5
```
Comment (jenniferboedker) [inline .github/release.yml:6]:
```suggestion
      - JohnnyQ5
      - github-actions
```

File: .github/workflows/label-pull-request.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: TimonVS/pr-labeler-action@v3
+        with:
+          configuration-path: .github/pr-labels.yml # optional, .github/pr-labeler.yml is the default value
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (jenniferboedker) [inline .github/workflows/label-pull-request.yml:15]:
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          
```

**Review by sven1103 — CHANGES_REQUESTED** [review]
The target branch needs to start with `ci/*` otherwise the auto-labeling wont do its correct job

**Comment by sven1103** [comment]
See above comment. Please checkout your changes to a branch called `ci/*` and open a new pr :)

**Comment by jenniferboedker** [comment]
see #90

**Pull request closed without merging**